src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -105,6 +105,9 @@ public function create(array $context = []): OpenApi
         return new OpenApi($info, $servers, $paths, new Model\Components(new \ArrayObject($schemas), new \ArrayObject(), new \ArrayObject(), new \ArrayObject(), new \ArrayObject(), new \ArrayObject(), new \ArrayObject($securitySchemes)), $securityRequirements);
     }
 
+    /**
+     * @return array | array
```

Comment:
`array | array` doesn't look right... maybe `array[]` was intended? or even `array{array, array}` (https://phpstan.org/writing-php-code/phpdoc-types#array-shapes)?

Reply:
indeed it should be array[]

Reply:
oh wait, I see that since then, `$schemas` has been [changed](https://github.com/api-platform/core/pull/4138/commits/e5cf715f8662509e0263cb859e27cb0070215863) from `array` to `\ArrayObject`... so that would give `array{array, \ArrayObject}` for PHPStan (or at this point maybe just remove this phpDoc?)

Reply:
actually the return has become useless, see #4199